File: exemplos/01/Apelidos.java
import java.util.ArrayList;
import static java.util.Arrays.asList;

public class Apelidos {
    public static void main(String[] args) {
        ArrayList<Integer> lista = new ArrayList<>(asList(10, 20, 30));
        int soma = 0;
        for (Integer x : lista) {
            soma += x;
            if (x == 20) {
                lista.add(1);
            }
        }
        assert soma == 61;
    }
}

File: exemplos/01/apelidos.cpp
#include <cassert>
#include <vector>

using namespace std;

int main(int argc, char *argv[]) {
    vector<int> v = {10, 20, 30};
    int soma = 0;
    for (int &x : v) {
        soma += x;
        if (x == 20) {
            v.push_back(1);
        }
    }
    assert(soma == 61);
    return 0;
}
